server/routes/statistic.js

```diff
@@ -8,9 +8,9 @@ module.exports = function initStatisticRoutes(app, conf) {
   //
   //// GET requests
 
-  app.get('/statistic', function (req, res) {
-    StatController.testStatistic();
-    res.render('statistic_main_view.ejs');
+  app.get('/statistic_home', function (req, res) {
+    var data = StatController.getAllStatistics();
```

Comment:
Following our coding style, the variable should be in camelCase. (StatController).